lib/instance.js

```diff
@@ -642,14 +643,14 @@ Instance.prototype.save = function(options) {
 
       if (self.isNewRecord) {
         query = 'insert';
-        args = [self, self.Model.getTableName(options), values, options];
+        args = [self, self.Model.getTableName(this.$options), values, options];
       } else {
         var where = this.where();
 
         where = Utils.mapValueFieldNames(where, Object.keys(where), this.Model);
 
         query = 'update';
-        args = [self, self.Model.getTableName(options), values, where, options];
+        args = [self, self.Model.getTableName(this.$options), values, where, options];
```

Comment:
Actually it is, we let users override it to provide user specific table names based on options passed to a call.
So it should still be `options`, not `this.$options`